apps/www/app/mail/_components/EmailViewer.tsx

```
@@ -0,0 +1,80 @@
+          </div>
+          <div className="flex items-center">
+            <span className="text-sm text-slate-500 mr-4">{email.timestamp}</span>
+            <FiCornerUpLeft className="mr-4 cursor-pointer" />
+            <FiCornerUpRight className="mr-4 cursor-pointer" />
+            <FiPrinter className="mr-4 cursor-pointer" />
+            <FiMoreVertical className="cursor-pointer" />
+          </div>
+        </div>
+        <h1 className="text-2xl font-bold mt-4">{email.subject}</h1>
+      </div>
+      <div className="flex-1 p-4 overflow-y-auto" dangerouslySetInnerHTML={{ __html: email.bodyContent }}></div>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)